config/samples/onload-module/mkdist-direct.Dockerfile

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: (c) Copyright 2023 Advanced Micro Devices, Inc.
+ARG DTK_AUTO
+ARG UBI_BASE=registry.access.redhat.com/ubi8/ubi-minimal:9.2
+
+
+FROM $DTK_AUTO as builder
+ARG ONLOAD_BUILD_PARAMS
+ARG ONLOAD_LOCATION
+ARG KERNEL_FULL_VERSION
+
+WORKDIR /build/
+ADD $ONLOAD_LOCATION onload.tar.gz
+RUN mkdir -p /build/onload
+RUN tar xzf onload.tar.gz -C /build/onload --strip-components=1
+WORKDIR /build/onload/
+
+ENV i_prefix=/opt
+RUN scripts/onload_build --kernel --kernelver $KERNEL_FULL_VERSION $ONLOAD_BUILD_PARAMS
+RUN scripts/onload_install --nobuild --kernelfiles --kernelver $KERNEL_FULL_VERSION
```

Comment:
I don't think this is sufficient. Specifically I think the `sfc.ko` and `sfc_driverlink.ko` kernel modules will fail to install when run in a cluster (see the comment in the [old sfc-module Dockerfile](https://github.com/Xilinx-CNS/kubernetes-onload/blob/46e1658e0abb0bb1a129832702dc3a70363ecf07/sfc/kmm/Dockerfile#L27), which still has to be used in #68 )

Reply:
Thank you Peter for filing https://jira.xilinx.com/browse/ON-15417. I suggest this thread can now be resolved and instead addressed as part of that new Jira ticket. What do you reckon?